### PR TITLE
Translate README.md to Traditional Chinese

### PR DESCRIPTION
**Pull request by qqaatw**
# What does this PR do?

1. Add README_zh-hant.md and links to direct users to each README.
2. Some of terms in the file can be found at [National Academy for Educational Research](https://terms.naer.edu.tw/), an official website providing bilingual translations between English and Traditional Chinese.

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

cc @JetRunner

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,11 @@ limitations under the License.
 </p>
 
 <h4 align="center">
-    <p><b>English</b> | <a href="https://github.com/huggingface/transformers/blob/master/README_zh-hans.md">简体中文</a><p>
+    <p>
+        <b>English</b> |
+        <a href="https://github.com/huggingface/transformers/blob/master/README_zh-hans.md">简体中文</a> |
+        <a href="https://github.com/huggingface/transformers/blob/master/README_zh-tw.md">繁體中文</a>
```
Comment (JetRunner) [inline README.md:45]: I recommend `zh-hans` and `zh-hant` instead of `zh-cn` and `zh-tw` to be more inclusive.

Comment (qqaatw) [reply README.md:45]: Sure, that makes sense.

**Comment by qqaatw** [comment]
I'll ask my friends who are also native in Traditional Chinese, to help double check the terms in the files. So, we can ensure the accuracy of the translation.

**Comment by qqaatw** [comment]
@JetRunner We have had the files checked. It is ready to merge if there is no other mistake.

**Comment by JetRunner** [comment]
Cool! I'll give it a look then we are ready to merge.

**Review by JetRunner — APPROVED** [review]
Super cool. Thanks @qqaatw and your friends!

**Pull request merged**